lib/ansible/playbook/block.py

```diff
@@ -370,7 +370,7 @@ def _get_parent_attribute(self, attr, extend=False, prepend=False):
     def filter_tagged_tasks(self, play_context, all_vars):
```

Comment:
remove play_context from the method's signature since it's no longer used.

Reply:
i kept for backwards compatibility, again its not needed and it is not really 'public api' but wanted to minimize breakage for others.